Firestore/Example/SwiftBuildTest/main.swift

```diff
@@ -217,6 +219,15 @@ func readDocument(at docRef: DocumentReference) {
   }
 }
 
+func readDocumentWithOptions(at docRef: DocumentReference) {
```

Comment:
Perhaps should now be named `readDocumentWithSource`?

Reply:
done